applications/dashboard/controllers/class.settingscontroller.php

```diff
@@ -558,6 +558,18 @@ public function branding() {
                 'Options' => [
                     'AllowEmpty' => true,
                 ]
+            ],
+            'Feature.DeferredLegacyScripts.Enabled' => [
+                'LabelCode' => t('Defer Javascript Loading'),
+                'Control' => 'toggle',
+                'Default' => true,
+                'Description' => t('This setting loads the page before executing Javascript which can improve your SEO.<br>
+                    <strong>**Warning: Enabling this feature may cause Javascript errors on your site.**</strong> ' .
+                    anchor(t('More information'), 'https://success.vanillaforums.com/entry/signin?Target=categories')),
```

Comment:
Is this supposed to be hidden behind a login?

Reply:
Also, can we get a `target="_blank"` on this link?

Reply:
https://success.vanillaforums.com/kb/articles/add?draftID=606